src/hint/HintTimer.js

```diff
@@ -0,0 +1,84 @@
+import { Debugger } from './../debug/Debugger';
+/**
+ *
+ * @export
+ * @class HintTimer
+ */
+export class HintTimer {
+  /**
+   * Creates an instance of HintTimer.
+   * @memberof HintTimer
+   */
+  constructor() {
+    this.listeners = [];
+  }
+
+  /**
+   * @param  {Number} time time in milliseconds
+   * @return {void}@memberof HintTimer
+   */
+  start(time) {
```

Comment:
Would it make sense to just have a default value instead of checking if a value was passed?

Reply:
I can do that.